docs/tutorials/github.md

```diff
@@ -54,43 +65,58 @@ You should now see the following directories in your repo.
 ```
 
-Our `profile.json` file contains a reference to our `catalog.json` file.  By default, this path is not resolvable by compliance-trestle, so we need to run the following command to update the `href` value in the JSON.
+3. Our `profile.json` file contains a reference to our `catalog.json` file.  By default, this path is not resolvable by compliance-trestle, so we need to run the following command to update the `href` value in the JSON.
 
 ```
 sed -i 's/NIST_SP-800-53_rev5_catalog.json/trestle:\/\/catalogs\/nist_rev5_800_53\/catalog.json/g' profiles/nist_rev5_800_53/profile.json
 ```
 
-Finally you can copy ready-made CI/CD workflows from the `TEMPLATES` directory into your workspace. These are the trestlebot actions that will run as we make changes to the repo contents.
+4. Ready-made CI/CD workflows can be copied from the `TEMPLATES` directory within the upstream `trestle-bot` repository into the local trestle workspace. These are the trestlebot actions that will run as changes are made to the repo contents.
+
+ * If trestlebot init was run earlier using a trestle-bot container image, then the upstream trestle-bot repository will first need to be cloned locally into a separate directory.
+```
+cd ..
+git clone https://github.com/RedHatProductSecurity/trestle-bot.git
+cd ../<trestle_workspace_repo>
+```
 
-**For example Component Definition authoring in GitHub Actions**
+ * Copy the required template workflows from the separate `trestle-bot` repository into the new workspace repository.
 ```
 mkdir -p .github/workflows
-cp TEMPLATES/github/trestlebot-create-component-definition.yml .github/workflows
-cp TEMPLATES/github/trestlebot-rules-transform.yml .github/workflows
+cp ../trestle-bot/TEMPLATES/github/trestlebot-create-component-definition.yml .github/workflows
+cp ../trestle-bot/TEMPLATES/github/trestlebot-rules-transform.yml .github/workflows
 ```
 
-Now that we have the initial content needed to begin authoring, go ahead and commit and push to the remote GitHub repo.
+5. Trestle-bot initial content is now created locally within the new trestle authoring workspace. This content can now be pushed to the remote GitHub repository.
+```
+git add .
+git commit -m "added example NIST SP 800-53 profile and component definition authoring workflow"
+git push
+```
+  *Note: if this is the first git push to the remote GitHub repository, then use `git push -u origin main` rather than `git push`.*
 
 
 ### 4. Create a New Component Definition
 
-Now it's time to run our first trestlebot action!  We will go ahead and create our first component definition.
+Now it's time to run our first trestlebot action within GitHub!  We will go ahead and create our first component definition.
 
-1. Open to your GitHub repo in a web browser.
+1. Open the remote workflow GitHub repository in a web browser.
```

Comment:
nitpick: maybe remove "workflow"?  Just to avoid any confusion that there may be more than 1 remote repo involved.